scripts/periodic_cleanup.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+# This script deletes instances whose name starts with "ra" if they are in
+# error state or older than AGE_LIMIT hours.
+
+
+import datetime
+import dateutil.parser
+from dateutil.tz import tzutc
+import os
+
+import pyrax
+
+
+def get_env_vars():
+    env_vars = {}
+
+    env_vars["age_limit"] = int(os.environ.get("INSTANCE_AGE_LIMIT", 48))
+    env_vars["instance_prefix"] = os.environ.get("INSTANCE_PREFIX", "ra")
```

Comment:
Doesn't this set env_vars["instance_prefix"] to "ra" if not set, which means the check below never fails?